pytorch/jobs/docker/1.9/py3/Dockerfile.gpu

```diff
@@ -136,28 +106,28 @@ RUN ${PIP} install --no-cache --upgrade \
         scikit-learn==0.20.3 \
         six==1.15.0 \
         scipy==1.7.3 \
-        torch==1.6.0 \
         typing_extensions==3.7.4.3
 
 RUN ${PIP} install --no-cache --upgrade sagemaker-training
 
-# Some packages can not be installed using pip (because they are not
-# python managed packages) - install them using apt get
-RUN apt-get update && apt-get install -y \
-        python-openbabel \
-        psi4
-
-RUN HOME_DIR=/root \
- && curl -o ${HOME_DIR}/oss_compliance.zip https://aws-dlinfra-utilities.s3.amazonaws.com/oss_compliance.zip \
- && unzip ${HOME_DIR}/oss_compliance.zip -d ${HOME_DIR}/ \
- && cp ${HOME_DIR}/oss_compliance/test/testOSSCompliance /usr/local/bin/testOSSCompliance \
- && chmod +x /usr/local/bin/testOSSCompliance \
- && chmod +x ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh \
- && ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR} ${PYTHON} \
- && rm -rf ${HOME_DIR}/oss_compliance*
+# install cuQuantum
```

Comment:
We should probably do this before we install the python libraries, but we can do in a future PR.